### ISSUE TITLE
Error running `make run_spin`

### ISSUE THREAD

**Issue by robotastic** [deislabs/containerd-wasm-shims#54]
When I run `make run_spin` in the top directory, everything builds correctly. However, when it gets to the install step, I get this error:
```bash
   Compiling containerd-shim-slight-v1 v0.1.0 (/home/luke/Projects/WASM/containerd-wasm-shims/containerd-shim-slight-v1)
    Finished release [optimized] target(s) in 10m 01s
sudo install target/release/containerd-shim-*-v1 /usr/local/bin
[sudo] password for luke: 
install: cannot stat 'target/release/containerd-shim-*-v1': No such file or directory
make: *** [Makefile:37: install] Error 1
```

That path does not existing in the top directory, but it is in `containerd-shim-spin-v1` and `containerd-shim-slight-v1`. 

It looks like install should add in the dir for the different shims: https://github.com/deislabs/containerd-wasm-shims/blob/4890298ea7acb8c208a2d117551b08ad23d4ea95/Makefile#L37

**Comment by Mossaka** [comment]
Oops, I think that might be my mistake when I removed workspace in #15 . Now, each spin and slight shim compiles within its own directory.

**Comment by Mossaka** [comment]
Should be fixed by #118
